includes/04-admin-menus.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Register admin menus and submenus for the plugin.
+ * Current pattern is to have only one top level menu 'UBC WP Vote' and all cpt and settings pages as its submenus.
+ *
+ * @package ubc_wp_vote
+ */
+
+namespace UBC\CTLT\WPVote;
+
+// If this file is called directly, abort.
+if ( ! defined( 'ABSPATH' ) ) {
+	die;
+}
+
+add_action( 'admin_menu', __NAMESPACE__ . '\\register_admin_menus' );
+
+/**
+ * Register admin menus
+ *
+ * @return void
+ */
+function register_admin_menus() {
+	// Main Menu.
+	add_menu_page(
```

Comment:
Do we absolutely need a top-level item here? If w're only having a settings page, we could perhaps either add a "Vote" submenu page to the settings section (alongside writing, reading, discussion etc.).